bin/ldapmanager

```diff
@@ -22,13 +30,12 @@ Example
     ./ldapmanager user u2
     ./ldapmanager user u3
 
-    ./ldapmanager passwd u1  # Requires input
-    ./ldapmanager passwd u1 --password secret
-    echo secret | ./ldapmanager password u1 -f-
+    ./ldapmanager passwd --login u1  # Requires input
```

Comment:
Exactly this line fails for me with 

```
./ldapmanager --bind-pass petrsown passwd --login mzeman
Password: 
Invalid Dn given : uid=mzeman,ou=Users,None (0x75 0x69 0x64 0x3D 0x6D 0x7A 0x65 0x6D 0x61 0x6E 0x2C 0x6F 0x75 0x3D 0x55 0x73 0x65 0x72 0x73 0x2C 0x4E 0x6F 0x6E 0x65 ) is invalid
Invalid DN syntax
```

Reply:
The ``--admin`` flag worked in all tests as expected, but see https://github.com/openmicroscopy/apacheds-docker/pull/9#discussion_r169443150 - is the dn creation in https://github.com/openmicroscopy/apacheds-docker/pull/9/files#diff-278610078cfba3d472f5f3bcbd71ff58R337 at fault ?

Reply:
Hmmm.... yes, I have a chicken and the egg problem. Working on a solution